lms/djangoapps/instructor/tasks.py

```diff
@@ -0,0 +1,96 @@
+""" Celery Tasks for the Instructor App """
+
+import logging
+
+from celery import shared_task
+from celery_utils.logged_task import LoggedTask
+from django.core.exceptions import ObjectDoesNotExist
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import UsageKey
+from xblock.completable import XBlockCompletionMode
+
+from common.djangoapps.student.models import get_user_by_username_or_email
+from lms.djangoapps.courseware.model_data import FieldDataCache
+from lms.djangoapps.courseware.module_render import get_module_for_descriptor
+from openedx.core.lib.request_utils import get_request_or_stub
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.exceptions import ItemNotFoundError
+
+log = logging.getLogger(__name__)
+
+
+@shared_task(base=LoggedTask, ignore_result=True)
+def complete_student_attempt_task(user_identifier, content_id):
+    """
+    Submits all completable xblocks inside of the content_id block to the
+    Completion Service to mark them as complete. One use case of this function is
```

Comment:
Sorry, now I'm just hyper aware of docstrings after talking about it already. Should have a summary line. And same String -> str comment below.